juce/loong_fix_relax.patch

```diff
@@ -0,0 +1,11 @@
+--- a/modules/juce_audio_devices/native/oboe/include/oboe/StabilizedCallback.h	2024-04-15 17:46:51.000000000 +0800
++++ b/modules/juce_audio_devices/native/oboe/include/oboe/StabilizedCallback.h	2024-11-27 19:40:34.884000000 +0800
+@@ -60,7 +60,7 @@
+ #if defined(__i386__) || defined(__x86_64__)
+ #define cpu_relax() asm volatile("rep; nop" ::: "memory");
+ 
+-#elif defined(__arm__) || defined(__mips__) || defined(__riscv)
++#elif defined(__arm__) || defined(__mips__) || defined(__riscv) || defined(__loongarch64)
```

Comment:
`__loongarch64` is deprecated. Read [this](https://wszqkzqk.github.io/2024/08/22/loongarchlinux-port-tips/#%E6%9C%89%E5%85%B3loongarch64%E7%9A%84%E7%BC%96%E8%AF%91%E5%99%A8%E9%A2%84%E5%AE%9A%E4%B9%89%E5%AE%8F).

Reply:
adjusted